src/libmongoc/tests/unified/result.c

```diff
@@ -338,8 +338,11 @@ result_check (result_t *result,
    bson_val_t *error_expect_result;
 
    if (!expect_result && !expect_error) {
-      /* TODO: "create index on a non-existing collection" seems to assume no
```

Comment:
When I originally added this TODO, I think I misinterpreted an "ns not found" failure in "create index on a non-existing collection" as meaning the operation was intended to fail.

Reply:
Ack.